tooling/miniutil/src/build/expr.rs

```diff
@@ -122,6 +122,13 @@ pub fn mul<T: TypeConv>(l: ValueExpr, r: ValueExpr) -> ValueExpr {
 pub fn div<T: TypeConv>(l: ValueExpr, r: ValueExpr) -> ValueExpr {
     int_binop::<T>(BinOpInt::Div, l, r)
 }
+pub fn bit_and<T: TypeConv>(l: ValueExpr, r: ValueExpr) -> ValueExpr {
```

Comment:
FYI: I ended up splitting this into 2 functions -- when something is 2 completely different operations in MiniRust, I don't think we want to hide that behind a single builder here.